code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -325,6 +325,7 @@
 /mob/living/silicon/robot/verb/Namepick()
 	set category = "Robot Commands"
 	if(custom_name)
+		to_chat(usr, "You can't pick another custom name. Go ask for a name change.")

```

Comment:
Missing filter tags, should be of the format `<span class='[label]'> [message]</span>`, where label is either a valid styling label (notice, warning, etc.,), or a filter label (filter_notice will send the messages to the same VChat filter as notice), so the messages get properly sorted by VChat.
The other change is also missing a label.